packages/ag-charts-community/src/chart/factory/expectedEnterpriseModules.ts

```diff
@@ -26,6 +26,7 @@ export const EXPECTED_ENTERPRISE_MODULES: EnterpriseModuleStub[] = [
         chartTypes: ['cartesian', 'polar', 'hierarchy'],
         identifier: 'gradient',
     },
+    { type: 'root', optionsKey: 'miniChart', chartTypes: ['cartesian'], identifier: 'mini-chart' },
```

Comment:
Nit: Drop `identifier`, it's mostly for series and axes cases where a `type` discriminator is involved (see other `'root'` cases above where we don't specify this too).